src/signalr/azext_signalr/tests/latest/test_signalr_commands.py

```diff
@@ -0,0 +1,80 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import time
+import unittest
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class AzureSignalRServiceScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(random_name_length=20)
```

Comment:
Can I know why is the length of the resource group name is limited to 20 instead of using the default value? Nothing wrong with configuring the value, but I'd like to know the motivation here.

Reply:
The default value (75) is so large that the deployment will fail because the RP seems can only accept the total length within 64.

Reply:
Cool.